packages/headless/src/app/engine-configuration-options.ts

```diff
@@ -40,6 +40,11 @@ export interface EngineConfigurationOptions {
    * The platformUrl() helper method can be useful to know what url is available.
    */
   platformUrl?: string;
+  /**
+   * The Engine name (e.g., myEngine). Specifying your Engine name will help in debugging when using an application with multiple Redux stores.
+   * @default 'coveo-headless'
```

Comment:
Even though `@default` provides syntax highlighting inside vscode, the rest of the project uses `@defaultValue` to follow the [TSDoc spec](https://tsdoc.org/pages/tags/defaultvalue/). I'd prefer to switch to stay consistent, because following the spec makes it easier to extract the documentation.